### ISSUE TITLE
Chatterino 2.5.2-beta.1 crash

### ISSUE THREAD

**Issue by ScrubN** [Chatterino/chatterino2#5770]
### Checklist

- [X] I'm reporting a problem with Chatterino
- [X] I've verified that I'm running **the most** recent nightly build or stable release
- [X] I've looked for my problem on the [wiki](https://wiki.chatterino.com/Help/)
- [X] I've searched the [issues and pull requests](https://github.com/Chatterino/chatterino2/issues?q=) for similar looking reports

### Describe your issue

I've been running the Chatterino beta for a few days, and last night it crashed while I was asleep.

[53d788c4-4fcc-420d-930f-29022b52b935.dmp.gz](https://github.com/user-attachments/files/18169185/53d788c4-4fcc-420d-930f-29022b52b935.dmp.gz)

### OS and Chatterino Version

Chatterino 2.5.2-beta.1 (commit 13d871fc) built with Qt 6.7.1, MSVC 194234435, Crashpad Running on Windows 10 Version 22H2, kernel: 10.0.19045

**Comment by ScrubN** [comment]
> Hmm, the crash only contains the exception handler in the stack and the stacks from other threads don't seem that interesting either.

Yeah, I didn't find anything of particular interest in windbg either.

I just realized that its possible the crash was not even related to Chatterino. I also had JetBrains Rider open at the time and running another project, although idling, and both of those also crashed overnight without any meaningful information in the Rider log nor Windows Event Viewer.

**Comment by ScrubN** [comment]
Got another crash that has a very similar stack trace, this time on 2.52.2. Still unsure if the cause was Chatterino or something with my system.

[d27256d4-d25b-4f79-9008-b0f246f6ee29.dmp](https://github.com/user-attachments/files/18324092/d27256d4-d25b-4f79-9008-b0f246f6ee29.dmp)

**Comment by ScrubN** [comment]
Another crash with another similar stack trace.

[b6281866-37fd-4112-a7c9-0241c32cca8c.dmp](https://github.com/user-attachments/files/18326438/b6281866-37fd-4112-a7c9-0241c32cca8c.dmp)

**Comment by pajlada** [comment]
You could try disabling the audio playback and see if the crash stops happening

In settings, find the "Sound backend" setting and change it from Miniaudio to Null

**Comment by ScrubN** [comment]
I've had the audio backend set to Null for 9 days now and no crashes. I'll try setting it back to Miniaudio and see if crashes resume.

**Comment by 8thony** [comment]
@ScrubN 
Since your last response, we updated Miniaudio (aee5f1a) and something changed with fore quit (0c489e0).
Additionally, 2.5.3 was released.

Does your crash still occur with "Sound backend" set back to Miniaudio?

**Comment by ScrubN** [comment]
Sorry for the delayed response.

I think the crashes were caused by an issue with Windows because they have not occurred again despite leaving my PC running with Chatterino open for almost 45 days. Because I left it running for so long though, I wasn't able to update to 2.5.3 when it released.

I have just updated to 2.5.3 though because Windows just hard crashed due to the long uptime 😅

**Comment by ScrubN** [comment]
Something else that may be worth mentioning, the audio stopped working for several applications during this experiment. The most notable ones I can remember would be the Windows device connect/disconnect sounds, Windows notifications, and Chatterino pings. Other applications like games, discord, or my browser continued to play audio as expected.

I'm not sure how helpful this is, but hopefully it's worth something?

**Comment by pajlada** [comment]
Marking this as fixed for now then. If you run into the same issue, please create a new issue and reference this one.
Thank you for taking time and effort into report & testing this!
